OJP/OJP_StopEvents.xsd

```diff
@@ -56,6 +56,11 @@
 					<xs:documentation>Transport operators to include/exclude.</xs:documentation>
 				</xs:annotation>
 			</xs:element>
+			<xs:element name="IncludeRestrictedLines" minOccurs="0">
+				<xs:annotation>
+					<xs:documentation>There might exist lines that have special restrictions and are not generally available to the public. E.g. school buses, company shuttles. dragLifts need to have an ACCESS MODE ski. Lines with ACCESS MODE bicycle will be included as well. If this flag is set, then existing restricted lines are considered by the router. The results are marked as restricted in the ServiceGroup. The restriction can be seen from: ACCESS MODE or PrivateMode. The usage could also be detailed with Attribute elements.</xs:documentation>
```

Comment:
Proposal:
There may be lines that are subject to special restrictions and are not generally open to the public. E.g. school buses, company shuttles. dragLifts must have an ACCESS MODE ski. Lines with ACCESS MODE bicycle are also considered. If this flag is set, existing restricted lines are considered by the router. The results are marked as restricted in the ServiceGroup. The restriction can be recognized by: ACCESS MODE or PrivateMode. The usage can also be detailed with attribute elements.

Reply:
changed